docs/pages/reference/helm-reference/teleport-plugin-mattermost.mdx

```diff
@@ -0,0 +1,251 @@
+---
+title: teleport-plugin-mattermost Chart Reference
+description: Values that can be set using the teleport-plugin-mattermost Helm chart
+---
+
+The `teleport-plugin-mattermost` Helm chart is used to configure the MatterMost Teleport plugin which allows users to receive access requests to channels or as direct messages via MatterMost.
+
+You can [browse the source on GitHub](https://github.com/gravitational/teleport-plugins/tree/master/charts/access/mattermost).
+
+This reference details available values for the `teleport-plugin-mattermost` chart.
+
+(!docs/pages/includes/backup-warning.mdx!)
+
+## `teleport.address`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+This parameter contains the host/port combination of the teleport auth server.
+
+<Tabs>
+  <TabItem label="values.yaml">
+  ```yaml
+  teleport:
+    address: "teleport.example.com:3025"
+  ```
+  </TabItem>
+  <TabItem label="--set">
+  ```code
+  $ --set teleport.address="teleport.example.com:3025"
+  ```
+  </TabItem>
+</Tabs>
+
+## `teleport.identitySecretName`
+
+| Type | Default value | Required? |
+| - | - | - |
+| `string` | `""` | Yes |
+
+Name of the Kubernetes secret that contains the credentials for the connection.
+
+The secret should be in the following format:
+
+```yaml
+apiVersion: v1
+kind: Secret
+type: Opaque
+metadata:
+  name: teleport-plugin-mattermost-identity
+data:
+  auth_id: ...
+```
+
+Check out the [MatterMost Helm Chart documentation](https://github.com/gravitational/teleport-plugins/tree/master/charts/access/mattermost#prerequisites) for more information about how to acquire these credentials!
```

Comment:
I think we should link to the Mattermost guide here so users stay within the docs site.